### ISSUE TITLE
Collapsible items on sidebar alignment issue

### ISSUE THREAD

**Issue by Ashwani44** [InfomediaLtd/angular2-materialize#271]
I am making a small demo in angular 2.  I am facing issue on my sidebar  alignment gets disturbed when i add Collapsible body . Please refer Image below .
here is my code : 
[code.txt](https://github.com/InfomediaLtd/angular2-materialize/files/1014776/code.txt)

The issue  i am getting is this : 
![issue](https://cloud.githubusercontent.com/assets/16367854/26255493/60f9ac1a-3cd7-11e7-89f4-4bec4b803da1.png)

**Comment by criticalbh** [comment]
Could you please provide plunker? Sharing code in txt file is not often the best practice.

**Comment by rubyboy** [comment]
@Ashwani44 does it not happen with plain MaterializeCSS ? (if you put the same code in a HTML file, which statically bring in MaterializeCSS)

**Comment by Ashwani44** [comment]
Yes, when i use Plain Materialize CSS  then my layout is proper, i don't find any issue but when i use  angular2-materialize then i get above issue ,my layout of collapsible gets disturbed.

here is my code  

        <div class="navbar-fixed"> 
        <nav class="blue">
        <div class="nav-wrapper">
         <a href="#!" class="brand-logo">
          <a href="#" data-activates="slide-out" materialize="sideNav" class="button-collapse show-on-large"><i class="material-icons">menu</i></a>Materialize</a>
        <ul class="right hide-on-med-and-down">
                 <li><a href="#">test1</a></li>
                  <li><a href="#">test1</a></li>
                   <li><a href="#">test1</a></li>
            </ul>
            
        </div>
    </nav>
    </div>     

        <ul id="slide-out" class="side-nav">
        <li><a href="#!"><i class="material-icons">dashboard</i>Dashboard</a></li>
       <li><a href="#!"><i class="material-icons">message</i>Messages</a></li>
       <li class="no-padding">
        <ul class="collapsible collapsible-accordion">
          <li>
            <a class="collapsible-header"><i class="material-icons">people</i>Friends</a>
            <div class="collapsible-body">
              <ul>
                <li><a href="#!">First</a></li>
                <li><a href="#!">Second</a></li>
                <li><a href="#!">Third</a></li>
                <li><a href="#!">Fourth</a></li>
              </ul>
            </div>
            </li>
          </ul>
         </li>
         
   
        </ul>

        
        <div class="container">
    <div class="row">
        <div class="col l9">
            <h3>Main</h3>

            <div class="divider"></div>
            <p>
            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
            </p>
        </div>
       
    </div>
</div>


My CSS 

        .side-nav {
         margin-top: 64.2px !important;
         }
        #sidenav-overlay{
        position:fixed;
        top:0;
        left:0;
        right:0;
        height:120vh;
        background-color:transparent!important;
       z-index:997;
       will-change:unset;
       }

**Comment by Syndlex** [comment]
Did you finde the Problem?

I'm use plain MaterializeCSS.

**Comment by Ashwani44** [comment]
Well i left using MaterializeCss as i was finding less support from
materialize as per angular 2 is concerned and i switched to angular 2
material ..i am doing good.
